common/models/model-definition.json

```diff
@@ -96,7 +96,8 @@
     "methods": {
       "embed": {
         "as": "object",
-        "key": "name"
+        "key": "name",
```

Comment:
I am confused. You can delete a property that does not exist:

```js
var data = {};
delete data.key;
// works fine, no error is thrown
```

I don't see why we need to keep the `key` around when there is `keyFn`, could you please check again?